ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -32,4 +32,18 @@ - (void)testFileFlagsBitIsSet {
     XCTAssertFalse(FileFlagsBitIsSet(FileFlags_IsInVirtualizationRoot, FileFlags_Invalid));
 }
 
+- (void)testShouldIgnoreVnodeType {
+    XCTAssertTrue(ShouldIgnoreVnodeType(VNON, NULL));
+    XCTAssertTrue(ShouldIgnoreVnodeType(VBLK, NULL));
+    XCTAssertTrue(ShouldIgnoreVnodeType(VCHR, NULL));
+    XCTAssertTrue(ShouldIgnoreVnodeType(VSOCK, NULL));
+    XCTAssertTrue(ShouldIgnoreVnodeType(VFIFO, NULL));
+    XCTAssertTrue(ShouldIgnoreVnodeType(VBAD, NULL));
+    XCTAssertFalse(ShouldIgnoreVnodeType(VREG, NULL));
+    XCTAssertFalse(ShouldIgnoreVnodeType(VDIR, NULL));
+    XCTAssertFalse(ShouldIgnoreVnodeType(VLNK, NULL));
+    XCTAssertFalse(ShouldIgnoreVnodeType(VSTR, NULL));
+    XCTAssertFalse(ShouldIgnoreVnodeType(VCPLX, NULL));
+}
```

Comment:
Also add a test that `ShouldIgnoreVnodeType` returns false for an unknown type?

Reply:
ShouldIgnoreVnodeType has takes in 'enum vtype' so I have to send that in.  I have a test case for each vtype.  Did you have something else in mind @wilbaker

Reply:
I was thinking something like:

```
// 10000 isn't found in the vtype enum
XCTAssertFalse(ShouldIgnoreVnodeType(static_cast<vtype>(10000), NULL));
```

To test this part of `ShouldIgnoreVnodeType`:

```
    default:
        KextLog_FileInfo(vnode, "vnode with unknown type %d encountered", vnodeType);
        return false;
```